packages/rancher-vsphere-csi/charts/templates/controller/deployment.yaml

```diff
@@ -39,25 +39,17 @@ spec:
                 values:
                 - "true"
       {{- end }}
+      {{- if .Values.csiController.tolerations }}
       tolerations:
-        # Rancher specific change: These tolerations are intentionally different from upstream to avoid lessening the scope to only NoSchedule with a specific key
-        # - key: node-role.kubernetes.io/master
-        #   operator: Exists
-        #   effect: NoSchedule
-        - operator: "Exists"
+      {{- with .Values.csiController.tolerations }}
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      {{- else }}
+      tolerations:
+        - key: node-role.kubernetes.io/master
+          operator: Exists
           effect: NoSchedule
-        - operator: "Exists"
-          effect: NoExecute
```

Comment:
Why is this one dropped?

Reply:
To match what's [upstream](https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/e878840857838f84e89784823cdbbd5d6b1c853a/manifests/vanilla/vsphere-csi-driver.yaml#L205-L218)

Reply:
It makes sense to me now.